app/shared/components/Global/Form/Field/MultiToken.js

```diff
@@ -14,16 +14,21 @@ export default class GlobalFormFieldMultiToken extends Component<Props> {
     };
   }
   onChange = debounce((e, { name, value }) => {
+    let { valid } = this.state;
+    if (name !== 'asset') {
+      valid = !!(value.match(/^\d+(\.\d{1,4})?$/g));
```

Comment:
This isn't valid - tokens can have more decimals than 4, or potentially even 0. I wouldn't try to validate the amount at all. With that being said - you can probably undo all the changes to this file.

Reply:
ok sounds good! I'll just do that 👍